src/EPR.Accreditation.Common.Data/Migrations/20240503060413_AccreditationFee.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EPR.Accreditation.API.Common.Data.Migrations
+{
+    public partial class AccreditationFee : Migration
```

Comment:
You need to remove this migration, get latest, then add your migration

Reply:
Done.